opensoundscape/ml/cnn.py

```diff
@@ -656,9 +666,9 @@ def _train_epoch(self, train_loader, wandb_session=None, progress_bar=True):
                 self._log(f"\tDistLoss: {epoch_loss_avg:.3f}")
 
                 # Evaluate with model's eval function
-                tgts = batch_labels.int().detach().cpu().numpy()
-                scores = logits.int().detach().cpu().numpy()
-                self.eval(tgts, scores, logging_offset=-1)
+                # tgts = batch_labels.detach().cpu().numpy()
+                # scores = logits.detach().cpu().numpy()
+                # self.eval(tgts, scores, logging_offset=-1)
```

Comment:
Can we note why these are commented out?

Reply:
we no longer use or report batch-level predictions and scores during the training loop, we just report the loss averaged